.env.production

```diff
@@ -0,0 +1,2 @@
+# We distribute this key to our clients, so checking this into git should be fine
+REACT_APP_MAPS_API_KEY=AIzaSyCjcq27gdshY3NVpOA5yuqvoQqML3YEQjI
```

Comment:
Kom noe mas fra GitGuardian, men mener dette ikke er noe stress

Reply:
Men for å slippe maset kan vi lage en ny nøkkel, legge den inn i secrets for repoet og slenge dette på bygg til prod-actionen:
```
    env:
      REACT_APP_MAPS_API_KEY: ${{ secrets.MAPS_API_KEY_PROD }}
```
. Jeg har ikke tilgang til å opprette secrets for repoet @magnusrand

Reply:
La den inn som en secret nå:) Navn: MAPS_API_KEY_PROD

Reply:
Hmm. Noe ble ikke helt riktig ser det ut til:
![image](https://user-images.githubusercontent.com/12783483/208290725-37906bfe-f331-4874-a80a-d1e101f563e4.png)


https://github.com/magnusrand/bryllup-anna-og-simen/actions/runs/3724192106/jobs/6316183281